routes/api/users.js

```diff
@@ -106,8 +107,9 @@ router.post('/users', validate, (req, res, next) => {
       { sendMail: verificationHelper.sendVerificationEmail(user), user }))
     .then(({ user }) => {
       const payload = {
-        username: user.username,
+        id: user.id,
```

Comment:
Since username is added for Oauth users as well then it would be good to include the username in the payload for consistency with locally registered user